app/services/external_api/va_notify_service.rb

```
@@ -33,20 +33,20 @@ class << self
     # Return: email_response: JSON response from VA Notify API
     # rubocop:disable Metrics/ParameterLists
     def send_email_notifications(
-      participant_id,
-      notification_id,
-      email_template_id,
-      first_name,
-      docket_number,
-      status = ""
+      participant_id:,
+      notification_id:,
+      email_template_id:,
+      first_name:,
+      docket_number:,
+      status: ""
```

Might be missing some ruby context but do these just default to `nil`? Should we specify that here?

---

Includes all other instances of this

---

I think we'd want to throw an error if a caller failed to provide any necessary information. I don't think there are solid default values for these, unfortunately.